src/containers/guest-home-page/signup-form/SignUpForm.jsx

```
@@ -6,12 +6,15 @@ import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import Checkbox from '@mui/material/Checkbox'
 import { FormControlLabel } from '@mui/material'
+import { useModalContext } from '~/context/modal-context'
```

ESLint

[error] 'useModalContext' is defined but never used.

(no-unused-vars)